various-receipes/db-optimization/docker-compose-mysql.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.6'
+services:
+  database:
+    image: mysql:8
+    command: --default-authentication-plugin=mysql_native_password --innodb_flush_log_at_trx_commit=2 --sync-binlog=0
```

Comment:
--innodb_flush_log_at_trx_commit=0 seems faster (not even writing immediately to log)

sync-binlog - Why not greater value like 20?

- Why not disable logging with --log-level=0? 

This comments applies to all DB 👆

Reply:
> --innodb_flush_log_at_trx_commit=0 seems faster (not even writing immediately to log)

Seems like you're correct

> sync-binlog - Why not greater value like 20?

0 disable the feature which is probably faster.

> Why not disable logging with --log-level=0

I think we already do so for pg, couldn't find something like this for MySQL